apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsDropdownControl.tsx

```diff
@@ -28,7 +28,12 @@ export function JsonFormsDropdownControl({
 }: ControlProps & OwnPropsOfEnum) {
   const [dropdownValue, setDropdownValue] = useState(data || "")
 
-  if (!options || (options.length === 1 && !!schema.default)) {
+  // Use the default value if it exists
+  useEffect(() => {
+    handleChange(path, schema.default || schema.const)
+  }, [path, schema.default, schema.const, handleChange])
+
```

Comment:
what do we feel about setting the default value in teh `useState`

Reply:
I think the `useState` not too important here cos the component is not rendered but I realised this also will cause changes even if the original data is defined. Fixed in 57174034e4c49db9f3c4d2359d82ad5f65bcc9cd.